### DISCUSSION TITLE
想知道对pdf ocr的功能比得上Adobe不，对pdf ocr后搜索的准确性

### DISCUSSION THREAD

**Discussion by wzgrx** [immersive-translate/immersive-translate#1967] [Help]
### 概述 | Summary

![Snipaste_2024-08-02_15-12-41](https://github.com/user-attachments/assets/a03ff26a-85ef-4399-9ddb-21ff6a71bde2)
自己只用得到ocr功能，Adobe ocr免费，但是有些识别不准确，漏字错字很多
上面的图片，包括文字+一个图片，文字一般都能ocr出来，但是图片里面的文字识别不了

想知道沉浸式翻译能做的更好吗？一个月69实在是太贵了，根本用不起

**Comment by wzgrx** [comment]
![Snipaste_2024-08-02_15-17-38](https://github.com/user-attachments/assets/90d09853-d6b9-4855-b9a6-248af8a30c26)
![Snipaste_2024-08-02_15-17-46](https://github.com/user-attachments/assets/1bbe6c02-9e37-4d78-a99b-26837d1a63ca)

**Comment by theowenyoung** [comment]
一般来说，像上图里的文字应该回被整体当成一个图片，不会被识别。您可以开通一个试用会员测试一下效果。
